lib/hydra/derivatives/runners/active_encode_derivatives.rb

```diff
@@ -26,5 +26,13 @@ def self.output_file_service
     def self.processor_class
       Processors::ActiveEncode
     end
+
+    def self.encode_class
```

Comment:
Do you think it would be better to pass the encode_class in through directives?  Then it could be configurable per call instead of only one active encode class at a time.

Hydra::Derivatives::ActiveEncodeDerivatives.create(video_record, encode_class: MyCustomEncode, source: :source_file_name, outputs: [low_res_video])